### ISSUE TITLE
[BUG] Angular material 16.2.0 breaks the sticky behavior of the search field

### ISSUE THREAD

**Issue by chutzemischt** [bithost-gmbh/ngx-mat-select-search#431]
**Describe the bug**
On opening the select field or scrolling inside the select, the search field is underneath the options.

**To Reproduce**
Using angular material version 16.2.x

[https://stackblitz.com/github/bithost-gmbh/ngx-mat-select-search-example](https://stackblitz.com/github/bithost-gmbh/ngx-mat-select-search-example?file=src%2Fapp%2Fapp.component.html)

**Expected behavior**
Search field stays on top and above the mat-option's

**Screenshots**
![image](https://github.com/bithost-gmbh/ngx-mat-select-search/assets/11884028/f3292712-9ea4-4224-bf6b-fd0eda027b40)

**Comment by macjohnny** [comment]
@chutzemischt thanks for reporting! Would you like to provide a fix?

**Comment by akaNightmare** [comment]
as I understand correctly the fix is
```
.mat-select-search-input {
    background-color: var(--mat-select-panel-background-color);
}
```

**Comment by sircrisp** [comment]
> as I understand correctly the fix is
> 
> ```
> .mat-select-search-input {
>     background-color: var(--mat-select-panel-background-color);
> }
> ```

this fixed it for me thanks

**Comment by macjohnny** [comment]
@akaNightmare thanks for the fix, it is released with [7.0.3](https://github.com/bithost-gmbh/ngx-mat-select-search/releases/tag/7.0.3)
